CppCoreGuidelines.md

```diff
@@ -6350,8 +6350,8 @@ Classes that represent exception objects need both to be polymorphic and copy-co
 
 ## C.other: Other default operation rules
 
-In addition to the operations for which the language offer default implementations,
-there are a few operations that are so foundational that it rules for their definition are needed:
+In addition to the operations for which the language offers default implementations,
+there are a few operations that are so foundational that specific rules for their definition are needed:
```

Comment:
Wasn't sure what exactly the latter wanted to say. But at least the original wording didn't appear to make sense...

Reply:
I think just removing "it" would have worked, or "explicit", but this is fine too. Thanks.